src/jwt.cpp

```diff
@@ -24,7 +24,7 @@
 static const String base64_chars =
     "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
     "abcdefghijklmnopqrstuvwxyz"
-    "0123456789+/";
```

Comment:
I think we want to keep the fixes that Alvaro patched in, I recommend removing this from your PR.

Reply:
yea i'll change it to Alvaros i thought i changed it on my fork but i guess not.